src/check_if_doc_exists.py

```diff
@@ -0,0 +1,42 @@
+import requests
+
+def check_if_doc_exists(params, headers, config):
+    """
+    Given
+    params:
+        doc_id - in format "DataSource:workspace_id:object_id"
+        index - elasticsearch index to search (required)
+        es_datatype - elasticsearch document data type
+    """
+    # verify inputs
+    if not params.get("doc_id"):
```

Comment:
Try out jsonschema for validating params. It's on my todos to add jsonschema validation for the other methods in this server.